### PR TITLE
fix input amount decimals, balance decimals, price representation

### PR DESCRIPTION
**Pull request by AuroraLantean**
- fix price representation in tokenList and orderList; rename consts.ts to settings.ts
- replaced the remaining toFixed() to formatPrice() on numeric display; 
- remove lg and most of console.log;

### CODE REVIEW

**Review by Warchant — APPROVED** [review]

File: src/app/bridge-btc/page.tsx [hunk 1]
```diff
@@ -346,39 +315,32 @@ const BridgeBtc = () => {
...
+			//console.log('tokens:', tokens)
 			setTokens(tokens);
 			const tokenObj = tokens.find((token) => token.symbol === tokenSymbol);
 			if (!tokenObj) {
 				console.error("No_token_found");
 				return;
 			}
 			setSelectedToken(tokenObj);
-			lg("tokenObj:", tokenObj);
+			console.log("tokenObj:", tokenObj);
```
Comment (uchu) [inline src/app/bridge-btc/page.tsx:328]: @AuroraLantean I don't see much value in having console logs in production at all.

File: src/app/bridge-btc/page.tsx [hunk 2]
```diff
@@ -346,39 +315,32 @@ const BridgeBtc = () => {
 	const [contract, setContract] = useState<Contract>(new Contract());
 	const [tokens, setTokens] = useState<Token[]>([]);
 	useEffect(() => {
-		lg("useEffect() ... setTokens");
 		const run = async () => {
 			const tokens = await contract.allTokensWithInfo();
-			lg("tokens:", tokens);
+			//console.log('tokens:', tokens)
```
Comment (uchu) [inline src/app/bridge-btc/page.tsx:320]: It should be either a console log or we don't need commented strings.

File: src/app/transfer/page.tsx [hunk 3]
```diff
@@ -306,12 +306,12 @@ export default function Transfer() {
 	}, 0);
 	const amountPrice =
 		modalAmount * (selectedToken ? prices.get(selectedToken.symbol) ?? 0 : 0);
-
+	//{formatter().format(total)}
```
Comment (uchu) [inline src/app/transfer/page.tsx:309]: Same, no need to commit commented lines.

File: src/services/utils.ts [hunk 4]
```diff
@@ -12,24 +12,55 @@ export const strToBn = (str: string): BN => {
...
 };
+export const formatter = (mfd = 2, currencyName = "usd") => {
+	let formatter: any;
+	switch (currencyName) {
+		case "usd":
+			formatter = new Intl.NumberFormat("en-US", {
+				style: "currency",
+				currency: "USD",
+				minimumFractionDigits: mfd,
+				//maximumSignificantDigits: 3
```
Comment (uchu) [inline src/services/utils.ts:23]: No need to commit commented. It should either here or not.

**Review by uchu — COMMENTED** [review]
@AuroraLantean Looks good. Will need to test it manually with more tokens and prices, but hopefully it will not reqiore significant changes.

Left just minor comments - no need to commit commented strings. And no need to use console.logs just for the sake of having them. Better to keep console clean as possible.

But maybe you can address that next time when you work on these files.

**Pull request merged**